src/objects/Hierarchy_Settings__c.object

```diff
@@ -15,6 +15,28 @@
         <unique>false</unique>
     </fields>
     <fields>
+        <fullName>Affl_ProgEnroll_Del__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Determines is an Affiliation will be automatically deleted when the related Program Enrollment is deleted. Otherwise the status will be changed to Former.</description>
```

Comment:
It's pulling it from the setting right below: Affl_ProgEnroll_Del_Status__c. We set it to "Former" during installation, but then the admin can change it.

Yep, that's a typo. I wanted to say "Determines if".